examples/D3D12Binding.cpp

```diff
@@ -152,14 +172,47 @@ class D3D12Binding : public BackendBinding {
         ComPtr<IDXGISwapChain3> swapChain;
         ComPtr<ID3D12DescriptorHeap> renderTargetViewHeap;
         ComPtr<ID3D12Resource> renderTargetResources[kFrameCount];
-        uint64_t fenceValues[kFrameCount] = { 0, 0 };
+        ComPtr<ID3D12CommandAllocator> commandAllocators[kFrameCount];
+        ComPtr<ID3D12GraphicsCommandList> commandList;
+        uint64_t fenceValues[kFrameCount] = {0};
```

Comment:
Can you also address this comment from the other review? Especially since you introduced a framenumber.

> I found the code dealing with fenceValues hard to follow. To make it a bit simpler, you could have only one fence-updated variable, and call it something like "lastFrameFinishedSerial". You'll need another variable (not used in a fence) that's "currentFrameSerial". What do you think?